content/page/what.en.md

```diff
@@ -73,33 +73,33 @@ If you are curious or want to help us to improve εxodus, you can file issues or
 
 <a name="android-app"></a>
 
-## εxodus Android application
+## εxodus Android Application
 
-This application is meant **to list the applications you have installed on your mobile phone** (from Google Play or F-Droid) and to tell you what are the trackers embedded in each application.
+Our application is designed **to list the applications already installed on your mobile phone** (both from [Google Play](https://play.google.com/store) or [F-Droid](f-droid.org/) and report the trackers embedded in each application.
 
 The application is available on [F-Droid](https://f-droid.org/packages/org.eu.exodus_privacy.exodusprivacy/) and [Google Play](https://play.google.com/store/apps/details?id=org.eu.exodus_privacy.exodusprivacy).
 
 <div class="row">
   <div class="col-md-6 text-center">
-  {{< fig src="/media/exodus-app-en-1.png" caption="Overview 1 of the εxodus Android application" >}}
+  {{< fig src="/media/exodus-app-en-1.png" caption="Overview 1 of the εxodus Android application - List of installed applications" >}}
   </div>
   <div class="col-md-6 text-center">
-  {{< fig src="/media/exodus-app-en-2.png" caption="Overview 2 of the εxodus Android application" >}}
+  {{< fig src="/media/exodus-app-en-2.png" caption="Overview 2 of the εxodus Android application - Tracker details for an application" >}}
   </div>
 </div>
 
-The application does not analyze the installed applications on you smartphone but retrieves the reports available on the [εxodus platform](https://reports.exodus-privacy.eu.org) and [does not contain any trackers](https://reports.exodus-privacy.eu.org/reports/search/org.eu.exodus_privacy.exodusprivacy).
+The application does not analyse the applications installed directly on your smartphone, instead it retrieves the reports already available on the [εxodus platform](https://reports.exodus-privacy.eu.org). Of course, the εxodus application itself [does not contain any trackers](https://reports.exodus-privacy.eu.org/reports/search/org.eu.exodus_privacy.exodusprivacy).
 
 <hr>
 
 <a name="videos"></a>
 
-## Popularization supports
+## Outreach: Learn More
 
-Exodus Privacy main interest is to make people aware of the tracking done by mobile applications. To do so, we are creating videos explaining how and why tracking massively occurs on smartphones. These videos are available on 2 platforms:
+The main goal of εxodus Privacy is to raise awareness of the extent of tracking by and in mobile applications. To this end, we have created videos explaining how and why tracking massively occurs on smartphones. You can watch these videos on 2 platforms:
```

Comment:
Same here, Exodus Privacy and not εxodus Privacy

Reply:
Got it! Changed.